script/VerifyMaxVault.s.sol

```diff
@@ -20,15 +20,21 @@ contract VerifyMaxVault is BaseScript, Test {
     }
 
     function run() public {
-        _loadDeployment();
         _setup();
-
+        _loadDeployment();
+        assertNotEq(msg.sender, deployer, "msg.sender should not be deploye as this is a verifier script.");
```

Comment:
this was added so that it doesn't accidentally inject the default foundry msg.sender in there which is what gets added normally

and it parses out the one from the json file.